### PR TITLE
Updating sidebar position in settings modal

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
Updating sidebar position in settings modal

#### Screenshots
|  before  |  after  |
|----|----|
| 
Screenshot 2023-12-15 at 7 23 17 PM
 | 
Screenshot 2023-12-15 at 7 21 29 PM
 |

-->

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by larkox** [comment]
The `position--top` class name is added on `src/components/settings_sidebar/settings_sidebar.tsx` lines between 56 and 60. If we are removing this style, almost sure we can remove this too. We would have to double check it doesn't break anything in the mobile view (the only other place where `position--top` is used apparently).

**Comment by asaadmahmood** [comment]
@larkox Removed it, and yup, we don't need it, its actually causing an issue now on firefox with the new modal updates.

**Comment by larkox** [comment]
@asaadmahmood Sorry for the back and forth with this :sweat_smile:  `position--top` is also defined on `src/sass/responsive/_mobile.scss` line 276. Just wanted to double check if this had to stay (and maybe the whole `.nav` class as you removed from `_settings.scss`) or not.

**Comment by asaadmahmood** [comment]
@larkox Yup, we don't need it. I checked mobile too.

**Review by larkox — APPROVED** [review]
Great work!

**Comment by larkox** [comment]
@asaadmahmood Heads up that there seems to be a merge conflict.

**Review by yasserfaraazkhan — APPROVED** [review]

File: webapp/channels/src/sass/routes/_settings.scss [hunk 1]
```diff
@@ -198,16 +198,6 @@
                 vertical-align: top;
             }
 
-            .nav {
-                position: fixed;
```
Comment (sinansonmez) [inline webapp/channels/src/sass/routes/_settings.scss:202]: @asaadmahmood we might still need this. If the content of the modal (i.e. right side of the modal) is scrollable, navigation also scrolls. If you agree, I can add back on my PR

**Pull request merged**